cpp/neuralnet/nninputs.h

```diff
@@ -31,6 +34,8 @@ struct MiscNNInputParams {
   double playoutDoublingAdvantage = 0.0;
   float nnPolicyTemperature = 1.0f;
   bool avoidMYTDaggerHack = false;
+  // If no symmetry is specified, it will use default or random based on config.
+  int symmetry = SYMMETRY_NOTSPECIFIED;
```

Comment:
Should mention that this value may be ignored even when specified, if there is a cache hit, if not also specifying skipCache?

Reply:
Note added